orchestrator/build.gradle

```diff
@@ -121,5 +155,6 @@ semanticRelease {
     repo {
         ghToken = project.ghToken
         releaseAsset distZip
+        releaseAsset createDeb.getOutputs().getFiles() - createDeb.getChangesFile()
```

Comment:
Unsure this is the best way to do it, if someone is better at gradle than me please feel free to advise! Basically the createDeb task creates two files, the deb and a changes file, but releaseAsset takes only one (and we probably only want the deb to be attached to the release)